domain-models-runtime/src/test/java/org/folio/rest/persist/PgUtilIT.java

```diff
@@ -1107,6 +1107,14 @@ public void optimizedSQLwithNo500(TestContext testContext) {
         }));
   }
 
+  @Test
+  public void deleteByCQLwithNo500(TestContext testContext) {
+    PgUtil.delete("users",  "username=delete_test ",
+        okapiHeaders, vertx.getOrCreateContext(), ResponseWithout500.class, testContext.asyncAssertFailure(e -> {
+          assertThat(e, is(instanceOf(NullPointerException.class)));
+        }));
+  }
```

Comment:
Unit tests for the other parts of the new code are missing.

Reply:
https://issues.folio.org/browse/DEBT-3 requests 11 APIs where this PgUtil.delete will be used, more to come.
Therefore we should have 100 % code coverage. Examples how to do this are the other unit tests in PgUtilIT.